src/iommu/iommufd.c

```diff
@@ -168,8 +168,7 @@ int vfio_get_device_fd(struct vfio_container *vfio, const char *bdf)
 
 close_dev:
 	/* closing devfd will automatically unbind it from iommufd */
-	if (close(devfd))
-		log_error("failed to close cdev fd: %d\n", devfd);
+	log_fatal_if(close(devfd), "close\n");
```

Comment:
Could we give a bit more information on the error? like the original message?

That goes for all the log_fata_if on this patch.

Reply:
Yeah, true. I will readd that.